pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -429,10 +429,10 @@ LANGUAGE SQL VOLATILE PARALLEL SAFE;
 
 --Returns the jsonb for a series defined by a label_array
 --Note that this function should be optimized for performance
-CREATE OR REPLACE FUNCTION SCHEMA_PROM.label_array_to_jsonb(labels int[])
-RETURNS jsonb AS $$
+CREATE OR REPLACE FUNCTION SCHEMA_PROM.label_array_to_kv(labels int[], OUT keys text[], OUT vals text[])
```

Comment:
This shouldn't replace the function but be in addition to. People querying from SQL may very well want to get jsonb out. Also note the inverse is called `key_value_array_to_label_array` we should be. consistent either way (don't feel strongly which)